docs/apis-tools/camunda-api-rest/intermediate-tutorial.md

```diff
@@ -0,0 +1,239 @@
+
+These keys will be consumed by the `auth.js` file to execute the OAuth protocol, and should be saved when you generate your client credentials in [prerequisites](#prerequisites).
+
+Examine the existing `.env.example` file for an example of how your `.env` file should look upon completion. Do not place your credentials in the `.env.example` file, as this example file is not protected by the `.gitignore`.
+
+:::note
+
+In this tutorial, we will execute arguments to deploy a resource, create and start a process instance, and view a process instance by its key. You can examine the framework for processing these arguments in the `cli.js` file before getting started.
+
+:::
+
+## Deploy resources (POST)
+
+First, let's script an API call to deploy a resource.
+
+To do this, take the following steps:
+
+1. In the file named `camunda-process-instances.js`, outline the authentication and authorization configuration in the first few lines. This will pull in your `.env` variables to obtain an access token before making any API calls:
+
+```javascript
+const authorizationConfiguration = {
+  clientId: process.env.CAMUNDA_CLIENT_ID,
+  clientSecret: process.env.CAMUNDA_CLIENT_SECRET,
+  // These settings come from your .env file. Note that CAMUNDA_TOKEN_AUDIENCE is represented by ZEEBE_TOKEN_AUDIENCE in the Console UI.
+  audience: process.env.CAMUNDA_TOKEN_AUDIENCE,
+};
+```
+
+2. Examine the function `async function deployResources()` below this configuration. This is where you will script out your API call.
+3. Within the function, you must first generate an access token for this request, so your function should now look like the following:
+
+```javascript
+async function deployResources() {
+  const accessToken = await getAccessToken(authorizationConfiguration);
+}
+```
+
+4. Using your generated client credentials from [prerequisites](#prerequisites), capture your Zeebe API URL beneath your call for an access token by defining `camundaApiUrl`:
+
+```javascript
+const camundaApiUrl = process.env.CAMUNDA_REST_ADDRESS;
+```
+
+5. On the next line, script the API endpoint to list the existing roles:
+
+```javascript
+const url = `${camundaApiUrl}/deployments`;
+```
+
+6. We will now configure the variables representing the BPMN file and its form data. This may look different depending on which resources you choose to deploy, but reflects the block-scoped local variables and append method to insert a set of objects for the BPMN resource of this tutorial:
+
+```javascript
+const formData = new FormData();
+// Read the BPMN file and add it to the form data
+const bpmnFilePath = path.resolve("resources/calculate-sales-tax.bpmn");
+const fileContent = fs.readFileSync(bpmnFilePath);
+formData.append("resources", fileContent, {
+  filename: "calculate-sales-tax.bpmn",
+  contentType: "application/xml",
+});
```

Comment:
non-blocking: there are a few bits of this code that would cause the upload to fail, without giving any sort of squiggly error in the editor: 

1. the name of the field on 93 must be `resources`, because the API requires it
2. The path to the file on 91 must be correct
3. The contentType on 95 must be `application/xml` (or at least, it should be, and I don't know what will happen if it isn't)

Is there anything we should add to specify these details?